src/multikzg/msm/mod.rs

```diff
@@ -0,0 +1,25 @@
+//! Utilities and tests for multi-scalar multiplication.
```

Comment:
can we try to get rid of this folder (which is copied from `gemini` already) and try to use the traits `VariableBaseMSM` that is implememented by any `CurveGroup`

Reply:
fixed